tests/nosetests/pyanaconda_tests/module_part_manual_test.py

```diff
@@ -78,93 +78,54 @@ def mount_points_property_test(self):
             []
         )
 
-        in_value = [
-            {
-                "mount-point": "/boot",
-                "device-spec": "/dev/sda1"
-            }
-        ]
-
-        out_value = [
-            {
-                "mount-point": get_variant(Str, "/boot"),
-                "device-spec": get_variant(Str, "/dev/sda1"),
-                "reformat": get_variant(Bool, False),
-                "format-type": get_variant(Str, ""),
-                "format-options": get_variant(Str, ""),
-                "mount-options": get_variant(Str, "")
-            }
-        ]
-
         self._test_dbus_property(
             "Requests",
-            in_value,
-            out_value
+            [
+                {
+                    "mount-point": get_variant(Str, "/boot"),
+                    "device-spec": get_variant(Str, "/dev/sda1"),
+                    "reformat": get_variant(Bool, False),
+                    "format-type": get_variant(Str, ""),
+                    "format-options": get_variant(Str, ""),
+                    "mount-options": get_variant(Str, "")
+                }
+            ]
```

Comment:
These and similar changes are not that self explanatory as the original variant. When you have `in_value` and `out_value` it's immediately clear what that means but adding `[ { ... } ]` does not say anything. Could you please add naming or comment?

Reply:
Fixed.